frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersionsTableRow.tsx

```diff
@@ -127,15 +135,16 @@ const ModelVersionsTableRow: React.FC<ModelVersionsTableRowProps> = ({
           ) : null}
           {isDeployModalOpen ? (
             <DeployRegisteredModelModal
-              onSubmit={() =>
+              onSubmit={async () => {
+                await handleAfterDeploy();
```

Comment:
I think you also need to add this to where DeployRegisteredModelModal gets rendered in ModelVersionDetailsHeaderActions